ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/LatLongAlt.java

```diff
@@ -14,24 +15,34 @@
      * Stores the altitude in meters.
      */
     private double mAltitude;
+    private Frame mFrame;
+
+    public LatLongAlt() {
```

Comment:
it looks like the above is related with change b343b8cd43410e28f74b809eeda31f2507621a54